frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureStrategyList/StrategyList/StrategyItem/PlaygroundStrategyExecution/PlaygroundStrategyExecution.tsx

```diff
@@ -79,5 +82,17 @@ export const StrategyExecution: FC<StrategyExecutionProps> = ({
         ),
     ].filter(Boolean);
 
-    return <ConstraintsList>{items}</ConstraintsList>;
+    const { variant, variants } = strategyResult.result as unknown as Partial<{
+        variant: PlaygroundFeatureSchemaVariant;
+        variants: StrategyVariantSchema[];
+    }>;
```

Comment:
Looks like we don't document everything we return as Playground Result from API. Variants are present in the response, but not in OpenAPI types

Reply:
Ah, good catch. is there a pr incoming for fixing the openapi docs too?